portal/eproms_substudy_tailored_content/src/components/Error.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+    <aside id="errorMessage" v-bind:class="{active: getAppObj().hasError()}">
+        <div class="body">
+            <!-- display generic message to user, but error details can be viewed in console -->
+            <div v-if="getAppObj().errorMessage" class="error-message content">Error retrieving data, see console for detail</div>
```

Comment:
Does this error message need to be translated, or is it only intended to be seen by devs?

Reply:
Yeah, I believe this will need to be translated. I think I will do a separate PR when the need arises. My understanding is that the current language for the sub-study is English for now - AU english, US english, Canadian english & UK english.